grails-app/services/org/broadinstitute/orsp/DataUseLetterService.groovy

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.orsp
+
+import grails.gorm.transactions.Transactional
+
+class DataUseLetterService {
+    QueryService queryService
+
+    String generateDul(Object input) {
```

Comment:
nit: It would prefer if we could have a better type for this. That would allow you to avoid things like `input["user"] as String`. Groovy lets you do this kind of dynamic typing, but it generally leads to bugs in the long run.